plugins/nginx-vhosts/templates/404-error.html

```diff
@@ -0,0 +1,9 @@
+<html>
+  <body style="padding:0; margin: 0;">
+    <div style="width:100%; height: 100%; text-align:center; font-family: arial; font-weight: bold; color: #ffffff; background: rgba(33, 129, 243, 0.66);">
+      <div style="position:absolute; top: 50%; left: 50%; transform: translateX(-50%) translateY(-50%);">
+        <h1>THERE HAS BEEN A 404 ERROR.</h1>
```

Comment:
For 404, maybe we can use `The page you were looking for doesn't exist.` `You may have mistyped the address or the page may have moved.` https://github.com/rails/rails/tree/master/actionpack/test/fixtures/public